components/pkg-export-docker/defaults/Dockerfile.hbs

```diff
@@ -1,6 +1,15 @@
 FROM {{base_image}}
 ENV PATH {{path}}
-ADD {{rootfs}} /
+
+ADD {{rootfs}}/root /root
+ADD {{rootfs}}/tmp /tmp
+ADD {{rootfs}}/var /var
+ADD {{rootfs}}/etc /etc
```

Comment:
Just for my own learning, why do we create these directories now? Just convenient for the user?

Reply:
We start from a `scratch` image, which has absolutely nothing in it. We've always created these directories (as well as several items inside `/etc`) to ensure things work properly.

This is just breaking it out explicitly.